lib/discourse.php

```diff
@@ -205,7 +207,7 @@ function sso_parse_request( $wp )
           'email' => $current_user->user_email,
           'about_me' => $current_user->description,
           'external_id' => $current_user->ID,
-          'avatar_url' => self::get_avatar_url($current_user->ID)
+          'avatar_url' => $avatar
```

Comment:
can you just make this `'avatar_url' => get_avatar_url(get_current_user_id())` and get rid of the new variables?